sky/packages/sky/lib/src/gestures/arena.dart

```diff
@@ -45,6 +45,7 @@ class GestureArenaEntry {
 class _GestureArenaState {
   final List<GestureArenaMember> members = new List<GestureArenaMember>();
   bool isOpen = true;
+  bool isHeld = false;
```

Comment:
Perhaps it's not valid to hold a closed arena, but you probably want to be able to close a held arena.  So I think state can be one of { open+released, open+held, closed+held, closed+released }. I'm not opposed to an enum but I'd like suggestions on what it would look like.